tests/atoms/condlink.scm

```diff
@@ -0,0 +1,59 @@
+
+(use-modules (opencog) (opencog exec))
+
+(define single
+	; In this flattened CondLink every item in even places is a condition
+	; for the next expression.
+	(CondLink
+	 ; If this evaluates to true then the next expresion will execute.
+	 (GreaterThanLink
+		(PlusLink
+		 (NumberNode 2)
+		 (NumberNode 2))
+		(TimesLink
+		 (NumberNode 3)
+		 (NumberNode 2)))
+	 ; Expression for the above condition.
+	 (PlusLink (Number 3)(Number 2))
+	 ; This is the second condition
+	 (FalseLink )
+	 ; Expression for the second condition.
+	 (PlusLink (Number 1)(Number 2))
+	 ; If all the above conditions evaluate to false, then this is the defualt
+	 ; expression to be executed.
+	 (PlusLink (Number 1)(Number -2)))
+)
+
+(define nondefault
+	; In this flattened CondLink every item in even places is a condition
+	; for the next expression.
+	(CondLink
+   ; If this evaluates to true then the next expresion will execute.
+		(GreaterThanLink
+		 (PlusLink
+			(NumberNode 2)
+			(NumberNode 2))
+		 (TimesLink
+			(NumberNode 1)
+			(NumberNode 2)))
+		(PlusLink (Number 3)(Number 2))
+		(FalseLink )
+		(PlusLink (Number 1)(Number 2)))
+)
+
```

Comment:
Is there an example of common usecase:
```
if (expression) 
     some code
else
      some other code
```
?

Reply:
do you mean like this?
<pre>
(CondLink
  |expression|                  ; EvaluatableLink
  |some-code|                  ; if |expression| evaluates to true
  |some-other-code|          ; else
)
</pre>
https://wiki.opencog.org/w/CondLink

Reply:
Yes